realtime_augmentation.py

```diff
@@ -0,0 +1,30 @@
+import cv2 as cv
```

Comment:
@mrshu I just didn't want to clutter up the utils file with these augmentation functions. But you might be right and probably should be merged with utils